openpype/settings/entities/schemas/projects_schema/schemas/schema_global_publish.json

```diff
@@ -267,13 +267,23 @@
                                     "type": "dict",
                                     "children": [
                                         {
-                                            "key": "output_extension",
-                                            "label": "Output extension",
+                                            "key": "extension",
+                                            "label": "Extension",
+                                            "type": "text"
+                                        },
+                                        {
+                                            "key": "colorspace",
+                                            "label": "Colorspace",
+                                            "type": "text"
+                                        },
+                                        {
+                                            "key": "display",
+                                            "label": "Display",
                                             "type": "text"
                                         },
                                         {
-                                            "key": "output_colorspace",
-                                            "label": "Output colorspace",
+                                            "key": "view",
+                                            "label": "View",
                                             "type": "text"
```

Comment:
Was the name/definition wrong originally? I don't believe **View** is equivalent to what could be an **Output Colorspace**?

From the [OCIO Glossary](https://opencolorio.readthedocs.io/en/latest/concepts/overview/overview.html#glossary):

- **Transform** - a function that alters RGB(A) data (e.g transform an image from scene linear to sRGB)
- **Reference space** - a space that connects colorspaces
- **Colorspace** - a meaningful space that can be transferred to and from the reference space
- **Display** - a virtual or physical display device (e.g an sRGB display device)
- **View** - a meaningful view of the reference space on a Display (e.g a film emulation view on an sRGB display device)
- **Role** - abstract colorspace naming (e.g specify the “lnh” colorspace as the scene_linear role, or the color-picker UI uses color_picking role)
- **Look** - a color transform which applies a creative look (for example a per-shot neutral grade to remove color-casts from a sequence of film scans, or a DI look)

Reply:
Sorry, you'll have to redirect to https://github.com/ynput/OpenPype/pull/4291